server/src/main/java/com/flipkart/varadhi/web/v1/TopicHandlers.java

```diff
@@ -49,15 +52,18 @@ public List<RouteDefinition> get() {
                 "/v1/tenants/:tenant/topics",
                 List.of(
                         new RouteDefinition(
-                                HttpMethod.GET, "/:topic", Set.of(), this::get,
+                                HttpMethod.GET, "/:topic", Set.of(), Sets.newLinkedHashSet(Sets.newHashSet()),
```

Comment:
Set.of() should be better instead of (Linked)HashSet() ?

Reply:
The set doesn't matter insertion order, If we say order has to maintain for handlers we need a data structure which remembers insertion